visualisation/visualise-submissions/visualise-submission-functions.R

```diff
@@ -0,0 +1,142 @@
+plot_forecasts = function(national = TRUE,
+                          cutoff = 0){
```

Comment:
lots and lots of repeated code between the raw and submission plots? Could be better to share some code.